.github/workflows/acceptance-tests.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v4
         with:
-          go-version: '1.18'
+          go-version: stable
```

Comment:
I haven't tested 1.20. Please bring back 1.19

Reply:
Then what it does? [Documentation](https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file) says you should use different option then. And for `go-version`:

> go-version: '1.18.0-rc.1' # The Go version to download (if necessary) and use.
> If stable is provided, action will get the latest stable version from the [go-versions](https://github.com/actions/go-versions/blob/main/versions-manifest.json) repository manifest.

Reply:
I have provided an answer to about this during our internal discussion, but here's a follow-up either way: changing the `go-version` from `1.18` to `stable` in this workflow doesn't actually change the version of Go being used—instead, it sets the Go version to be the latest `stable` release, while still respecting the [Go 1 compatibility promise](https://go.dev/doc/go1compat)—this means that any valid Go program written under Go 1.x should continue to work without modification under any future Go 1.y release; additionally, packages and programs compiled with earlier versions of Go will continue to work with future versions of the Go compiler as the Go project has made a commitment to maintain the stability of the language and its standard library, so developers can rely on their code continuing to work as expected even as new versions of Go are released

Reply:
changed to `go-version-file: go.mod`